src/main/java/AgeVerification.java

```diff
@@ -0,0 +1,11 @@
+public class AgeVerification {
+
+    public static boolean isAdult(int a) {
+        if (a>=18) {
```

Comment:
takie porównanie samo w sobie zwraca typ boolean więc wystarczy zrobić
`return a >= 18;`
Twoje rozwiązanie nie jest złe, natomiast można napisać to zwięźlej.

Reply:
Poprawione,